src/shogun/statistical_testing/MultiKernelQuadraticTimeMMD.h

```diff
@@ -87,7 +87,7 @@ class CMultiKernelQuadraticTimeMMD : public CSGObject
 	 *
 	 * @param kernel The kernel instance.
 	 */
-	void add_kernel(CShiftInvariantKernel *kernel);
+	void add_kernel(CKernel *kernel);
```

Comment:
@lambday it would be so great if we could port the stat testing stuff to the new API ... it causes problems all the time. I think everything to do that is in place now, it is really just a matter of refactoring the code

Reply:
@karlnapf I'll get in touch with you about this soon. I can manage some time for Shogun now after work.

Reply:
That’d be ace!